### ISSUE TITLE
Add description and recommendation about unicode LTR and RTL marks

### ISSUE THREAD

**Issue by andoks** [mdn/mdn#540]
### Proposal

Add description and recommendation about unicode LEFT-TO-RIGHT MARK ([U+200E](https://unicode-explorer.com/c/200E)) and RIGHT-TO-LEFT MARK ([U+200F](https://unicode-explorer.com/c/200F)) to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdo.

- when to use
- pros and cons vs <bdo> and <bdi>

### Browser support

Firefox

### Tasks

- [ ]

### Dependencies

- [ ]

### Are you willing to support this work?

Can provide reviews - but my motivation for the topic is that I myself am not a subject matter expert, and maybe this would fit on MDN

**Comment by pepelsbey** [comment]
You’re right! Unfortunately, there is currently no RTL/LTR guide that combines all methods for working with different writing directions. Such a guide would be a perfect place to mention Unicode symbols and accessibility considerations.
